livelossplot/plot_losses.py

```diff
@@ -11,13 +11,17 @@ class PlotLosses:
     """
     Class collect metrics from the training engine and send it to plugins, when send is called
     """
-    def __init__(self, outputs: Optional[List[Type[BO]]] = None, **kwargs):
+    def __init__(self, outputs: Optional[List[Type[BO]]] = None, mode: str = 'notebook', **kwargs):
         """
         :param outputs: list of callbacks (outputs) which are called with send method
+        :param mode: mode for callbacks - some of outputs need to change some behaviors,
+         depending on the working environment (scripts and jupyter notebooks)
```

Comment:
Still, it needs to be explicit. `Options: 'notebook' or 'script'` or something in that line.

Reply:
oki